CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.3.19
```

Comment:
Please remove this version and add changes to the 0.3.18.

Reply:
> Please remove this version and add changes to the 0.3.18.

Thanks for the review comments @tunacicek . I have made the changes.